packages/client/package.json

```diff
@@ -63,12 +65,12 @@
     ]
   },
   "devDependencies": {
+    "@mdx-js/mdx": "^1.6.22",
```

Comment:
If we're not using mdx let's get rid of it.
```suggestion
```

Reply:
Not certain on the mdx, will follow-up with the team though so this is omitted if unused

@3dvkr @TelescopeThieves do either of you know if we still have plans to use this? TY!

Reply:
MDX confirmed as unnec, removed from client pkg